dspace-api/src/main/java/org/dspace/handle/dao/impl/HandleDAOImpl.java

```diff
@@ -83,7 +83,7 @@ public long countHandlesByPrefix(Context context, String prefix) throws SQLExcep
     @Override
     public int updateHandlesWithNewPrefix(Context context, String newPrefix, String oldPrefix) throws SQLException
     {
-        String hql = "UPDATE handle set handle = concat(:newPrefix,'/',id WHERE handle like :oldPrefix%";
+        String hql = "UPDATE Handle set handle = concat(:newPrefix,'/',handle_id) WHERE handle like concat(:oldPrefix,'%')";
```

Comment:
Aha, I think @abollini is correct.  This is HQL and not SQL. And, Hibernate does prefer using the name of the *entity* (in this case "id") instead of the name of the actual DB column (in this case "handle_id").  So, it probably is more correct to use "id" here.